src/workers/parser/getSymbols.js

```diff
@@ -159,10 +160,12 @@ function extractSymbols(source: Source) {
 
       if (t.isCallExpression(path)) {
         const callee = path.node.callee;
+        const args = path.node.arguments;
         if (!t.isMemberExpression(callee)) {
           const { start, end, identifierName } = callee.loc;
           callExpressions.push({
             name: identifierName,
+            arguments: args,
```

Comment:
Lets add this to the formatFixtures function so we can see the output.

Reply:
Where is the `formatFixtures` function?

Reply:
oops: https://github.com/devtools-html/debugger.html/blob/master/src/workers/parser/utils/formatSymbols.js#L23